Add iOS Page (#770)
privacyguides/privacyguides.org: docs/ios/overview.md

---
title: iOS
---
iOS is the operating system that runs on iPhones. iOS is a good choice if you want privacy and security without much effort.

## App Privacy Report

App Privacy Report is a built-in tool that allows you to see what permissions your apps are using.

## Siri

> If you want to make use of Apple's digital assistan, be aware that Siri collects speech data,[^1] although Apple claims that only minimal data is sent.

Reviewer: ```suggestion
If you want to make use of Apple's digital assistan, be aware that Siri collects speech data ,[^1] although Apple claims that only minimal data is sent and as of iOS 15, Siri uses Machine Learning and totally uses Voice Data on Device to process things.
```

Reviewer: As it is totally On-Device, I think you should phrase it as, iOS sends data back to apple only when you opt-in for analytics or Speech Improvement services.

> If you want to make use of Apple's digital assistan, be aware that Siri collects speech data,[^1] although Apple claims that only minimal data is sent.

Reviewer: ```suggestion
If you want to make use of Apple's digital assistant, be sure to make it only activate after the Device is unlocked to prevent unauthorized access via Utlrasonic Hacking techniques.

As of iOS 15, iOS uses Machine Learning & On-Device Proccessing to deliver things and your Vocie data never leaves your device. Unless you opt-in(or Opt-out **Not sure**) for Analytics or Speech Improvement services.
```

> [^1]: [Ask Siri, Dictation & Privacy](https://www.apple.com/legal/privacy/data/en/ask-siri-dictation/)

Reviewer: ```suggestion

```

> If you want to make use of Apple's digital assistan, be aware that Siri collects speech data, [^1] although Apple claims that only minimal data is sent.

Reviewer: ```suggestion
If you want to make use of Apple's digital assistant, be aware that Siri collects speech data, [^1] although Apple claims that only minimal data is sent.
```
